app/assets/js/components/Work/Tabs/Preservation/FileSetModal.jsx

```diff
@@ -17,11 +17,25 @@ import WorkTabsPreservationFileSetForm from "@js/components/Work/Tabs/Preservati
 import classNames from "classnames";
 import useAcceptedMimeTypes from "@js/hooks/useAcceptedMimeTypes";
 import { useCodeLists } from "@js/context/code-list-context";
+import S3ObjectPicker from "@js/components/Work/Tabs/Preservation/S3ObjectPicker"
 
 const modalCss = css`
   z-index: 100;
 `;
 
+const sectionHeaderCss = css`
+  font-size: 1.2rem;
+  font-weight: bold;
+  margin-bottom: 1rem;
+`;
+
+const sectionCss = css`
+  margin-bottom: 2rem;
+  padding: 1rem;
+  border: 1px solid #ddd;
+  border-radius: 4px;
+`;
+
```

Comment:
As @adamjarling noted, we can probably lean on existing Meadow and bulma classes for this.

```suggestion
```

Reply:
Just pushed a commit to replaced this with the `box` class, thanks @mathewjordan !